server/routes/fixtures/SuperTestWrapper.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import request from 'supertest'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock'
+import express from 'express'
+import app from '../../index'
+import config from '../../config'
+
+jest.mock('redis', () => jest.requireActual('redis-mock'))
+
+export default class SuperTestWrapper {
```

Comment:
This is the main class you are going to use when writing supertest tests - basically new up a `SuperTestWrapper` and use it's `request` property to issue requests.
It also has helper methods to make the test context authenticated or unauthenticated (need to add a method to authenticate as a mail room user, but that will come ....)